stories/ui/Badge.js

```diff
@@ -18,10 +18,11 @@ const types = [
   'requires-payment',
 ];
 
-export type Props = {|
+export type Props = {
   props: BadgeProps,
-|};
+};
 
+// $FLOW_FIXME: I made PropsMatrixType exact but it's still not working here :/
```

Comment:
I got stuck on this flow issue here (same thing in Button). I thought if I made this not an exact flow type, the `wrapperClassName` wouldn't be an issue (wrapperClassName is needed for the Rating component). Please let me know if you have any suggestions?